### PR TITLE
Removed the name and default path elements from XProj factory registration

### PR DESCRIPTION
**Pull request by 333fred**
This fixes the XProj Migration factory showing up in the new project dialog. Addresses #466.

Tagging @dotnet/project-system for review.

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx [hunk 1]
```diff
@@ -130,12 +130,9 @@
     <value>A project for creating a command-line application that can run on .NET Core on Windows, Linux and MacOS.</value>
   </data>
   <data name="5" xml:space="preserve">
-    <value>XProj Files (*.xproj);*.xproj</value>
```
Comment (333fred) [inline src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx]: @srivatsn, I removed the xproj resource in VSResources.resx and changed the other two numbers to be sequential with existing stuff. Is that going to break anything?

Comment (srivatsn) [inline src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx]: Yes the C# and VB [templates](https://github.com/dotnet/sdk/blob/master/src/Templates/ProjectTemplates/VisualBasic/.NETCore/VisualBasicClassLibrary/ClassLibrary.vstemplate) need to be adjusted.

**Comment by srivatsn** [comment]
👍 once you fix up the templates.

File: src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx (hunk 1 above)
Comment (333fred) [inline src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx]: Submitted dotnet/sdk#134

**Comment by 333fred** [comment]
@dotnet-bot retest Windows Release please

#431.

**Pull request merged**